project/MimaSettings.scala

```diff
@@ -11,22 +11,24 @@ import com.typesafe.tools.mima.plugin.MimaKeys.{previousArtifacts, reportBinaryI
  */
 object MimaSettings {
 
-  // The `previousVersion` must be exactly the previous version (e.g. "2.3.1" for "2.3.2-SNAPSHOT").
+  // The `previousVersions` must be the *ALL* previous versions (e.g. Set("2.3.0", "2.3.1") for "2.3.2-SNAPSHOT").
   //
   // The following bad scenario is the reason we must obey the rule:
   //
-  //  - your build is toward 2.3.2 release and the `previousVersion` is "2.3.0"
+  //  - your build is toward 2.3.2 release and the `previousVersions` is "2.3.0" only
   //  - you've added new methods since 2.3.1
   //  - you're going to remove some of the methods in 2.3.2
-  //  - MiMa cannot detect such bin-incompatibility
+  //  - cannot detect such bin-incompatibility
```

Comment:
If you remove the subject here, how about `in this case, the incompatibily won't be detected`?